Index: templates/sign_disp.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html>\r\n<head>\r\n    <title>Registration Form</title>\r\n</head>\r\n<body>\r\n    <h1>Registration Form</h1>\r\n\r\n    <form action=\"/register\" method=\"POST\">\r\n        <input name=\"full_name\" type=\"text\" placeholder=\"Full Name\" required>\r\n        <input name=\"username\" type=\"text\" placeholder=\"Username\" required>\r\n        <input name=\"password\" type=\"password\" placeholder=\"Password\" required>\r\n        <input type=\"submit\" value=\"Register\">\r\n    </form>\r\n\r\n    {% if error_message %}\r\n    <p>{{ error_message }}</p>\r\n    {% endif %}\r\n\r\n    <h2>List of Users</h2>\r\n    <ul>\r\n        {% for user in users %}\r\n            <li>{{ user.full_name }} - {{ user.username }}</li>\r\n        {% endfor %}\r\n    </ul>\r\n</body>\r\n</html>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/sign_disp.html b/templates/sign_disp.html
--- a/templates/sign_disp.html	(revision 0050065290fbad446aac14703529b8decb239c28)
+++ b/templates/sign_disp.html	(date 1695300772334)
@@ -6,12 +6,12 @@
 <body>
     <h1>Registration Form</h1>
 
-    <form action="/register" method="POST">
-        <input name="full_name" type="text" placeholder="Full Name" required>
-        <input name="username" type="text" placeholder="Username" required>
-        <input name="password" type="password" placeholder="Password" required>
-        <input type="submit" value="Register">
-    </form>
+<!--    <form action="/register" method="POST">-->
+<!--        <input name="full_name" type="text" placeholder="Full Name" required>-->
+<!--        <input name="username" type="text" placeholder="Username" required>-->
+<!--        <input name="password" type="password" placeholder="Password" required>-->
+<!--        <input type="submit" value="Register">-->
+<!--    </form>-->
 
     {% if error_message %}
     <p>{{ error_message }}</p>
